### ISSUE TITLE
Mage T9 2piece bonus

### ISSUE THREAD

**Issue by RandomXD** [dalaranwow/dalaran-wow#5638]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
If you equip 2 pieces of tier 9 on mage you will gain 15% of spirit as crit which is all good but once you unequip them and use other gear the 15% spirit as crit still stays
**Expected behaviour**: Tell us what should happen instead

**Steps to reproduce the problem**:

1. check the crit u have with ur normal gear
2. equip 2 pieces of t9 and use molten armor
3. put back ur normal gear and see how the crit increased cuz the 15% spirit as crit is still up

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by Chenocide** [comment]
Would like to see a source on this issues. 
Noticed that you don't really provide sources on any of your reports.

**Comment by Blackoutx** [comment]
pretty sure this wasn't changed until 4.0.1
